src/include/btmem.h

```diff
@@ -1622,6 +1629,17 @@ struct __wt_update {
     uint8_t data[]; /* start of the data */
 };
 
+static inline uint64_t
+get_upd_start_ts(WT_UPDATE *upd)
```

Comment:
After looking at this change, I feel we should always set prepare_ts to prepare_ts. Otherwise the logic is too complicated for other code. Also if we need to first read prepare state, we need to use a barrier, which will add more overhead. @tdlinh2712 Sorry for realizing this later. @kommiharibabu What do you think?

Reply:
You mean we should assign `prepare_ts` to both `prepare_ts` and `start_ts`? That's the latest behaviour after merging https://github.com/wiredtiger/wiredtiger/pull/12186.
I agree that this is quite invasive and if we do this, we would need to always requires a barrier everytime this function is called